modules/matriculaciones/routes/turno.ts

```diff
@@ -3,19 +3,21 @@ import { Auth } from '../../../auth/auth.class';
 import { Profesional } from '../../../core/tm/schemas/profesional';
 import * as turno from '../schemas/turno';
 import { turnoSolicitado } from '../schemas/turnoSolicitado';
+import { userScheduler } from '../../../config.private';
 const router = express.Router();
 
-router.post('/turnos/save/:turnoId', (request, response, errorHandler) => {
-
-    turno.findByIdAndUpdate(request.params.turnoId, request.body, { new: true }, (err, res) => {
-        if (err) {
-            return errorHandler(err);
+router.patch('/turnos/save/:turnoId', Auth.authenticate(), async (request, response, next) => {
```

Comment:
Ya que modificás las ruta, sacaría el `save` del path, q no es muy restful q digamos... el save ya está implicito el método patch
```suggestion
router.patch('/turnos/:turnoId', Auth.authenticate(), async (request, response, next) => {
```

Reply:
Concuerdo, pero dejemoslo para otra ocación. Ya es bastante larga la instancia de testing para esta tarea.